### PR TITLE
WIP: Add flutter packages get support

### PR DESCRIPTION
**Pull request by rrousselGit**
This PR adds support for `mono_repo pub get` on packages that depend on Flutter. 

Tests are missings, but the implementation work.
I do not have a windows computer with me though, nor is the CI configured to run tests on windows.

### CODE REVIEW

**Comment by rrousselGit** [comment]
@kevmoo Any idea how I can add tests for windows? 
This is a feature that will be needed for https://github.com/mobxjs/mobx.dart

**Comment by kevmoo** [comment]
For windows, we'd have to use https://www.appveyor.com/ – we've set this up on pkg:build for instance - https://github.com/dart-lang/build/blob/master/appveyor.yml

I wouldn't worry about windows testing here – it's more of a general issue we should consider with mono-repo (although your diligence is appreciated!)

File: mono_repo/lib/src/commands/pub.dart [hunk 1]
```diff
@@ -41,17 +41,29 @@ class _PubSubCommand extends MonoRepoCommand {
 }
 
 Future<Null> pub(String pubCommand, RootConfig rootConfig) async {
-  var pkgDirs = rootConfig.map((pc) => pc.relativePath).toList();
-
   print(lightBlue
-      .wrap('Running `pub $pubCommand` across ${pkgDirs.length} package(s).'));
+      // `pub` isn't really a fitting name anymore. What do we use?
```
Comment (kevmoo) [inline mono_repo/lib/src/commands/pub.dart]: We should move the `executable` check above this and then use executable + arguments in the content of the log

Comment (kevmoo) [reply mono_repo/lib/src/commands/pub.dart]: Put a more generic message outside the loop, then include the detailed command in the "Starting `$dir`... bit...

**Comment by kevmoo** [comment]
Status here?

**Comment by rrousselGit** [comment]
Oh, sorry about that.

I don't need it anymore. I made my own implementation of the package that better fits my needs

**Comment by kevmoo** [comment]
Okay. Will close this out!

**Pull request closed without merging**